models/keyword_test.go

```diff
@@ -99,6 +99,27 @@ var _ = Describe("Keyword", func() {
 				Expect(keywords[0].Id).To(Equal(keyword.Id))
 			})
 
+			Context("given an operator with case insensitive contain `keyword`", func() {
+				It("returns only `keyword` filter records", func() {
+					containKeyword := "keyword_keyword"
+					var queryList = map[string]interface{}{"keyword__icontains": containKeyword}
+
+					user := FabricateUser(faker.Email(), faker.Password())
+					firstKeyword := FabricateKeyword(faker.Word(), faker.URL(), 0, user)
+					secondKeyword := FabricateKeyword(faker.Word(), faker.URL(), 0, user)
+					thirdKeyword := FabricateKeyword(fmt.Sprintf("%v %v %v", faker.Word(), containKeyword, faker.Word()), faker.URL(), 0, user)
```

Comment:
Same as above. Think of using variabls which can help to make sense of all the fixtures

```suggestion
					matchedKeyword := FabricateKeyword(fmt.Sprintf("%v %v %v", faker.Word(), containKeyword, faker.Word()), faker.URL(), 0, user)
```